test/configs/prefetch_all_parquet_files.json

```diff
@@ -0,0 +1,8 @@
+{
+  "description": "Debug setting around testing of parquet files",
+  "skip_compiled": "true",
+  "on_init": "INSTALL parquet FROM 'build/release/repository'; LOAD parquet; SET prefetch_all_parquet_files=true;",
```

Comment:
Maybe we can add a new setting to avoid needing to do this, e.g.:

`"extensions": ["parquet"]`

We already have `core_functions` that we always load in `SQLLogicTestRunner::LoadDatabase`